src/js/disability-benefits/routes.jsx

```diff
@@ -12,6 +12,10 @@ import AdditionalEvidencePage from './containers/AdditionalEvidencePage.jsx';
 import ClaimEstimationPage from './containers/ClaimEstimationPage.jsx';
 
 const routes = [
+  <Redirect
```

Comment:
Did you test this to see that it works? It's a bit surprising to me that it would work, since the base url is set to `track-claims` and is prepended to urls in the route config here.